### DISCUSSION TITLE
Welcome to trafficbot Discussions!

### DISCUSSION THREAD

**Discussion by lucasveneno** [lucasveneno/trafficbot#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nbitelecom** [comment]
Hi this is Nikhil Bajaj . want to discuss Further and Explore opportunity to monetize the same i have Resources like proxies and all i need some solution in respect to bandwidth consumption and one Click and one hit on a page and scroll Upto 2 -3 pages
